content/en/docs/Writing policies/generate.md

```diff
@@ -246,7 +246,107 @@ spec:
           foo: bar
 ```
 
-## Generating resources into existing namespaces
+## Generate for Existing resources
+
+With Kyverno 1.7.0+, Kyverno supports the generate for existing resources. Generate existing policies are applied in the background which creates target resource in existing matched resources in the cluster.
+They may also optionally be configured to apply upon updates to the policy itself.
+
+### Generate NetworkPolicy on Existing Namespaces
+
+By default, policy will not be applied on existing trigger resource when it is installed. This behavior can be configured via `generateExistingOnPolicyUpdate` attribute. Only if you set `generateExistingOnPolicyUpdate` to `true`, Kyverno will generate the target secret resource in existing namespaces on policy CREATE and UPDATE events.
```

Comment:
Suggest removing specific Kind in this statement:

Kyverno will generate the target ~~secret~~ resources for existing triggers on ...